docs/modules/Record.ts.md

```diff
@@ -238,28 +386,63 @@ Added in v2.6.5
 
 ## difference
 
+Difference between 2 `Record`s.
+Takes 2 `Record`s and produces a `Record` composed by the
+entries of the two inputs, removing the entries with the same
+key in both inputs.
```

Comment:
Switching between "2" and "two" here is unexpected

Reply:
Clear. I'm replacing every occurrence in the file for consistency.
See commit ff55d3c.

Reply:
Thank you.

Reply:
It looks like currently things are still a mix between various function documentations.
My personal preference would be to use the word `two` and align with [common writing style guides](https://apastyle.apa.org/style-grammar-guidelines/numbers/words), but not a blocker by any means.